src/executor/execution_plan.cc

```diff
@@ -17,6 +17,8 @@ std::vector<T> filter(const std::vector<T>& vec, Predicate pred) {
 
 auto getOpType = [](const std::string& str) {
   if (str == "nop") {
+    return mscclpp::OperationType::THREADBLOCK_BARRIER;
```

Comment:
`"nop"` returns `THREADBLOCK_BARRIER`?

Reply:
Fixed. Return NOP instead of THREADBLOCK_BARRIER.